libcontainer/cgroups/fs/memory.go

```diff
@@ -140,23 +115,14 @@ func (s *MemoryGroup) Set(path string, cgroup *configs.Cgroup) error {
 		return err
 	}
 
-	if cgroup.Resources.KernelMemory != 0 {
-		if err := setKernelMemory(path, cgroup.Resources.KernelMemory); err != nil {
-			return err
-		}
-	}
+	// ignore KernelMemory and KernelMemoryTCP
```

Comment:
Can we at least print something if kernel memory limit is set so users can be aware of this?

Reply:
This is already done in specconv.

Reply:
https://github.com/opencontainers/runc/pull/2840/commits/2f54a31e3ec5461bf11d955ddb6927ab7a328952#diff-93ae3e89134c5b7d0b37869c5b950405c369d60559b300ed579b85b9f89f3884R256

Reply:
Isn't that only for update command?

Reply:
Yep I got it all mixed up; sorry about that.

Please see updated patch. In short, we
 - ignore kmem settings in cgroup v1 driver (same as we were doing for cgroup v2);
 - warn about settings being ignored in `specconv` and `runc update`.